test/e2e/framework/deployment/wait.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
last nit: 2019, cause new file :)

Reply:
fixed.
I think I got you mean now. It's not necessarily that the code came from a file that already existed but how Git sees it. i.e., fixtures.go is a new version of deployment_utils.go but wait.go is a new file from Git's point of view?

Reply:
yes! you can use `git mv` to control this better if you want